src/mpid/ch4/netmod/include/netmod.h

```diff
@@ -48,6 +48,7 @@ typedef int (*MPIDI_NM_am_isend_reply_t) (MPIR_Context_id_t context_id, int src_
                                           int handler_id, const void *am_hdr, size_t am_hdr_sz,
                                           const void *data, MPI_Count count, MPI_Datatype datatype,
                                           MPIR_Request * sreq);
+typedef int (*MPIDI_NM_am_recv_t) (MPIR_Request * rreq);
```

Comment:
We only call this function for the case when `MPI_{I}Recv` matches unexpected "ZCOPY" message, right? So I think the name is not precise. How about `MPIDI_NM_am_zcopy_start`?

Reply:
Sure. Let me change that.

Reply:
Actually, I think I will take that back. `am_recv` would be a much appropriate name because CH4 does not know what is ZCOPY. it just knows that "I have a request that has not started the receive yet".

Reply:
While in the current situation this only happens when ZCOPY data transmission is involved at the transport level, it does not mean the CH4 would knowingly call the "ZCOPY" interface.